docs/_source/guides/llms/conceptual_guides/rm.md

```diff
@@ -121,10 +120,24 @@ for record in prompts:
 
 # Add records to the dataset
 dataset.add_records(records)
+```
+
+::::{tab-set}
 
-# This publishes the dataset and pushes the records into Argilla
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+# This publishes the dataset with the records into Argilla and returns the dataset in Argilla
+remote_dataset = dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")
```

Comment:
The variable `remote_dataset` is not used in the rest of the code snippets in the page, so maybe `dataset.push_to_argilla()` is enough in this case?

Reply:
Even though it may not be used later, I think it's nice to show that they need to capture the return object, as the `warning` will be removed in the next release, and that affects users if they copy-paste the snippets pretending to use those further

Reply:
I agree with Alvaro. Even if it's not used, it's good to have it there so the user knows that the methods returns something that can be used.